### PR TITLE
[65] Event page

### PR DESCRIPTION
**Pull request by josephjyang**
Make it generic. Will need to connect to backend.

Ask maple for questions

### CODE REVIEW

File: frontend/src/pages/events/index.js [hunk 1]
```diff
@@ -7,29 +7,42 @@ import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEve
...
+    imgUrl: "../../images/prayer-gathering.png",
+    imgAlt: "example",
+    location: "TCenter",
+    description: "event at Hmcc"
+  }] // Todo: fetch actual events from backend
+
+  return (
+    <Layout>
+      <SundayCelebBarEvents />
+      <div>Event list section</div>
```
Comment (xumaple) [inline frontend/src/pages/events/index.js:24]: Can you make this into a different component (like `SundayCelebBarEvents`, `Instagram`) and put it into the components page?

File: frontend/src/pages/events/index.js [hunk 2]
```diff
@@ -7,29 +7,42 @@ import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEve
 import { StaticImage } from "gatsby-plugin-image";
```
Comment (xumaple) [inline frontend/src/pages/events/index.js:7]: add a comment with screenshots of desktop and mobile views so we can review more easily

File: frontend/src/pages/events/index.js (hunk 1 above)
Comment (xumaple) [reply frontend/src/pages/events/index.js:24]: and can put the `const events` list into the new component as well; we'll work on connecting to the backend soon.

**Review by xumaple — CHANGES_REQUESTED** [review]

File: frontend/src/pages/events/index.js [hunk 3]
```diff
@@ -6,33 +6,86 @@ import Instagram from "../../components/instaBar";
 import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEvents";
 import { StaticImage } from "gatsby-plugin-image";
 import EventCard from "../../components/page-events/eventCard";
-import Banner from "../../components/shared/banner";
+// import Banner from "../../components/shared/banner";
```
Comment (thedomcab) [inline frontend/src/pages/events/index.js:9]: Why did you comment out Banner?

Comment (terrianx) [reply frontend/src/pages/events/index.js:9]: it was not being used so i commented it. do you want me to uncomment it?

Comment (thedomcab) [reply frontend/src/pages/events/index.js:9]: based upon the changes you made it is being used: 
You remove the Banner Component for the EventPage.  Look at the Code you removed and replaced with.

Comment (terrianx) [reply frontend/src/pages/events/index.js]: hi dom ive fixed this issue and i think it should be able to be merged into main

**Review by thedomcab — APPROVED** [review]

**Review by xumaple — APPROVED** [review]

**Pull request merged**